### PR TITLE
Add secondary quad9 server

### PR DESCRIPTION
**Pull request by loganaden**
**By submitting this pull request, I confirm the following (please check boxes, eg [X]) _Failure to fill the template will close your PR_:**

***Please submit all pull requests against the `development` branch. Failure to do so will delay or deny your request***

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [X] I have checked that [another pull request](https://github.com/pi-hole/pi-hole/pulls) for this purpose does not exist.
- [X] I have considered, and confirmed that this submission will be valuable to others.
- [X] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [X] I give this submission freely, and claim no ownership to its content.

**How familiar are you with the codebase?:**

1

---
Add secondary quad9 server

_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._

### CODE REVIEW

File: automated install/basic-install.sh [hunk 1]
```diff
@@ -809,6 +809,7 @@ setDNS() {
     Quad9)
       echo "Quad9 servers"
       PIHOLE_DNS_1="9.9.9.9"
+      PIHOLE_DNS_2="149.112.112.112"
```
Comment (Mcat12) [inline automated install/basic-install.sh]: Where is this IP documented? I can't find it on their website.

Comment (loganaden) [reply automated install/basic-install.sh]: https://twitter.com/Quad9DNS/status/931618659407159297

Comment (Mcat12) [reply automated install/basic-install.sh]: Until they advertise it on their website, I'm unsure about adding it here.

Comment (loganaden) [reply automated install/basic-install.sh]: No worries, we can wait until their website is properly updated.

Comment (WaLLy3K) [reply automated install/basic-install.sh]: If it's advertised on their primary Twitter account, then that's pretty dang official. I'm okay with adding this, personally.

Comment (PromoFaux) [reply automated install/basic-install.sh]: Hmmm, there is a possibility that the secondary isn't even filtered, if [this conversation](https://github.com/Nyr/openvpn-install/issues/379) is anything to go by

**Comment by dschaper** [comment]
We recieved official confirmation that the .112.112 DNS service is Quad9. Would someone please contact Quad9 support and confirm if .112 a filtered DNS server or just a resolving server?

**Comment by loganaden** [comment]
I will contact them and ask.

**Comment by loganaden** [comment]
It is filtered.

**Comment by DL6ER** [comment]
Kicked codacy...

**Pull request merged**